tools/mo/openvino/tools/mo/analysis/json_print_new_frontend.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from openvino.runtime import PartialShape, Model, Type        # pylint: disable=no-name-in-module,import-error
+from openvino.runtime.utils.types import get_dtype
+
+def json_model_analysis_dump(framework_model: Model):
+
+    def dump_partial_shape(shape: PartialShape):
+        if shape.rank.is_dynamic:
+            return 'None'
+        return [dim.get_length() if dim.is_static else 0 for dim in shape]
```

Comment:
can just `str(shape)` be used to get string representation of shape?

Reply:
In general, yes, but we (rather) have to be the same as the old path, where `0` is printed as a dynamic dimension (in `str(shape)` we have `?` as dynamic dim)